unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/RPCSocialApiBridge.cs

```diff
@@ -87,17 +89,52 @@ public async UniTask<FriendshipInitializationMessage> GetInitializationInformati
 
             initializationInformationTask = new UniTaskCompletionSource<FriendshipInitializationMessage>();
             await InitializeMatrixTokenThenRetrieveAllFriends(cancellationToken);
+            var friendshipInitializationMessage = await GetRequestEvents(cancellationToken);
 
-            initializationInformationTask.TrySetResult(new FriendshipInitializationMessage
-            {
-                totalReceivedRequests = 0,
-            });
+            initializationInformationTask.TrySetResult(friendshipInitializationMessage);
 
             await UniTask.SwitchToMainThread(cancellationToken);
 
             return await initializationInformationTask.Task.AttachExternalCancellation(cancellationToken);
         }
 
+        private async UniTask<FriendshipInitializationMessage> GetRequestEvents(CancellationToken cancellationToken = default)
```

Comment:
in my opinion, the naming of this method doesn't represent the implementation details

Reply:
Renamed to `GetFriendRequestsFromServer`